<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/bin/conda-lint", line 11, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/run.py", line 107, in main
    result = linter.lint(
             ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 620, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 692, in lint_file
    messages = self.lint_recipe(
               ^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 748, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 247, in run
    self.check_recipe(recipe)
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/check_build_help.py", line 555, in check_recipe
    o = int(path.split("/")[1])
        ^^^^^^^^^^^^^^^^^^^^^^^
ValueError: invalid literal for int() with base 10: 'host'

ERROR conda.cli.main_run:execute(47): `conda run conda-lint /tmp/abs_cb_pmb76ro/clone` failed. (See above for error)
</details>

--- a/recipe/build_script.bat
+++ b/recipe/build_script.bat
@@ -1,1 +1,0 @@
-%PYTHON% -m pip install . --no-deps --no-build-isolation -vv

--- a/recipe/build_script.sh
+++ b/recipe/build_script.sh
@@ -1,4 +1,0 @@
-#!/usr/bin/env bash
-set -eux
-
-$PYTHON -m pip install . --no-deps --no-build-isolation -vv

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -40,7 +40,7 @@ outputs:
       run:
         - python
         - deprecated >=1.2.13
-        - numba>=0.53
+        - numba >=0.53
         - numpy >=1.21.0,<1.25
         - pandas >=1.1.0,<2.0.0
         - scikit-learn >=0.24.0,<1.3.0

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -11,6 +11,8 @@ source:
 
 build:
   number: 0
+  # numba is not available on s390x
+  skip: true  # [s390x]
   skip: true  # [py<37 or py>311]
 
 # This section is required to correctly rerender skip statement
@@ -96,10 +98,10 @@ outputs:
         - filterpy
         - h5py
         # - hmmlearn  # not on CF
-        # - gluonts  # not on CF
+        - gluonts
         - matplotlib-base
         - mne
-        - numba >=0.53                           # [py<311]
+        - numba >=0.53
         - pmdarima
         - prophet
         - pykalman
@@ -115,8 +117,6 @@ outputs:
         - xarray
 
     test:
-      requires:
-        - pip
       imports:
         - sktime
         - sktime.classification

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -22,21 +22,17 @@ requirements:
     - pip
     - setuptools
     - wheel
-    - toml
-    - python-build
 
 outputs:
   - name: sktime
-    script: build_script.sh  # [unix]
-    script: build_script.bat  # [win]
+    build:
+      script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
     requirements:
       host:
         - python
         - pip
         - setuptools
         - wheel
-        - toml
-        - python-build
       run:
         - python
         - deprecated >=1.2.13

